crates/astria-sequencer/src/bridge/bridge_lock_action.rs

```diff
@@ -226,18 +246,118 @@ mod tests {
 
         // enough balance; should pass
         let expected_deposit_fee = transfer_fee
-            + get_deposit_byte_len(&Deposit {
+            + calculate_base_deposit_fee(&Deposit {
                 bridge_address,
                 rollup_id,
                 amount: 100,
                 asset: asset.clone(),
                 destination_chain_address: "someaddress".to_string(),
                 source_transaction_id: transaction_id,
                 source_action_index: 0,
-            }) * 2;
+            })
+            .unwrap()
+                * 2;
         state
             .put_account_balance(from_address, &asset, 100 + expected_deposit_fee)
             .unwrap();
         bridge_lock.check_and_execute(&mut state).await.unwrap();
     }
+
+    #[test]
+    fn test_calculate_base_deposit_fee() {
```

Comment:
You are in a test so don't call the test a test:
```suggestion
    fn calculated_fee_matches_expected_value() {
```

I'd also refactor the test to use `#[track_caller]`. See my comment for testing `display_len` in astria_core.